testing/functional/model/dataGrid.ts

```diff
@@ -11,6 +11,7 @@ const CLASS = {
     groupRow: 'dx-group-row',
     commandEdit: 'dx-command-edit',
     commandExpand: 'dx-command-expand',
+    selectCommand: 'dx-command-select',
```

Comment:
Rename selectCommand to commandSelect like commandEdit/commandExpand

Reply:
Done.